db-test/tests/db/db/utilities.rkt

```diff
@@ -0,0 +1,10 @@
+#lang racket/base
+
+(provide (all-defined-out))
+
+;; Exn:fail -> Boolean?
+;;
+;; Receives an exn:fail exception, performs a regex match on its message and returns true
+;; if the exception does not represents the multiple statements given error.
+(define (exn:fail-is-not-multiple-statements-error? exn-fail)
```

Comment:
Suggestion: move this right before the relevant tests (instead of having it in a separate file)